src/sqlfluff/dialects/dialect_athena.py

```diff
@@ -244,6 +243,61 @@
 )
 
 
+class ArrayTypeSegment(ansi.ArrayTypeSegment):
+    """Prefix for array literals specifying the type."""
+
+    type = "array_type"
+    match_grammar = Sequence(
+        Ref.keyword("ARRAY"),
```

Comment:
Nit but doesn't require `Ref.keyword`

```suggestion
        "ARRAY",
```